verticapy/plotting/_highcharts/acf.py

```diff
@@ -46,7 +46,7 @@ def _init_style(self) -> None:
                 "title": {"text": "lag"},
                 "categories": self.data["x"].tolist(),
             },
-            "yAxis": {"title": {"text": "value"}, "max": 1},
```

Comment:
leave the max 1, ACF plots can not reach more than 1 and -1

Reply:
@mail4umar please tell me why you deleted the max?

Reply:
In order not to cut-off the plot to display. But since >1 is not expected so i will revert.